cmd/app/internal/service/tts-stt/tts-stt.py

```
@@ -91,50 +142,176 @@ async def speech_to_text(file: UploadFile):
+
+    final_wav = f"/tmp/{uuid.uuid4()}_final.wav"
+    final_mp3 = final_wav.replace(".wav", ".mp3")
+    concat_list = f"/tmp/{uuid.uuid4()}_list.txt"
+
+    with open(concat_list, 'w') as f:
+        for temp_file in temp_files:
+            f.write(f"file '{temp_file}'\n")
+
+    await asyncio.get_event_loop().run_in_executor(
+        executor,
+        lambda: subprocess.run([
```

Ruff (0.12.2)

`subprocess` call: check for execution of untrusted input

(S603)